tools/biohansel/biohansel.xml

```diff
@@ -331,8 +365,8 @@ Contents of ``match_results.tab``:
 
 
 
-Analysis of a single FASTQ readset
-----------------------------------
+**Example analysis results of a single FASTQ readset**
```

Comment:
Could you include a description of what the `subtype_metadata` file should look like someplace in the documentation here?

Reply:
I've added a description and example of a subtype metadata table in e3a1125fab6f8363a56f38ffc8eb461ce0029e51

Reply:
Awesome, thanks Peter.